src/openfl/display/SimpleButton.hx

```diff
@@ -101,6 +101,15 @@ class SimpleButton extends InteractiveObject {
 	}
 	
 	
+	private override function __forceRenderDirty ():Void {
+		
+		super.__forceRenderDirty();
+		
+		__currentState.__forceRenderDirty();
+		
+	}
```

Comment:
Many of the `SimpleButton` functions handle the `_currentState`, but also the `hitTestState` and check first if the states exist. See `setTransformDirty` as an example: https://github.com/innogames/openfl/blob/d8115491b63763ca63c8727c01f3851f5f898917/src/openfl/display/SimpleButton.hx#L378

Should we do it in a similar way?

Reply:
When you look at the code it at first sight, it looks like it can never be null. There also methods that are using it  without null checks. But current state can be null. Will update the code